packages/extensions/venia-adobe-data-layer/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@magento/venia-adobe-data-layer",
+  "version": "0.0.1",
+  "publishConfig": {
+    "access": "public"
+  },
+  "description": "Provides Adobe Client Data Layer support in PWA Studio",
+  "main": "./index.js",
+  "scripts": {
+    "clean": " "
+  },
+  "repository": "github:magento/pwa-studio",
+  "license": "(OSL-3.0 OR AFL-3.0)",
+  "dependencies": {
+    "@adobe/adobe-client-data-layer": "~1.1.3"
+  },
+  "peerDependencies": {
```

Comment:
I saw you were importing `react in the `index.js` file. Shouldn't you add that as a peer dep?

Reply:
Good catch. Will add it.